src/internal/utilities.ts

```diff
@@ -2,7 +2,7 @@ import * as types from '../types';
 import { Dictionary } from './formatting';
 
 /** Raf for node + browser */
-export const raf = typeof requestAnimationFrame === 'undefined' ? setTimeout : requestAnimationFrame.bind(window);
+export const raf = (win: Window | undefined) => typeof requestAnimationFrame === 'undefined' ? setTimeout : requestAnimationFrame.bind(win);
```

Comment:
can this be memoized some way? Otherwise we need to re-bind `requestAnimationFrame` every time `_styleUpdated` is called.

Reply:
I'm in favor of removing .bind and just doing feature detection each time it is called.  In order words, wrapping this in a function and checking window for rAF each time rAF is called.  Since this does not get called very often, this should not pose any real performance issue

Reply:
You could also get rid of the `.bind` by doing
```typescript
const raf = typeof requestAnimationFrame === 'undefined' ? f => setTimeout(f) : f => requestAnimationFrame(f)
```